### DISCUSSION TITLE
Running Directus in a subdirectory on a webserver

### DISCUSSION THREAD

**Discussion by navanshu** [directus/directus#17261] [Archive]
Hi I am running directus on our website, using the latest container must be (9.22.4) with minio and postgis
The container is working fine and I have provided its a `PUBLIC_URL` and able to reach it from nginx acting as a reverse proxy on `url/directus/` and it send me to `url/directus/admin`

![image](https://user-images.githubusercontent.com/7664426/213865746-5536dcb7-48a4-4cbe-89f6-0d63affe19e6.png)

Though the manifest file is loading, but I am not able to get the page working
![image](https://user-images.githubusercontent.com/7664426/213865785-361473b4-859a-4821-8010-9798d4e93813.png)

Is there a workaround for this bug, the container logs show no information, it just says Get /admin
![image](https://user-images.githubusercontent.com/7664426/213865863-d7c377d1-5dc0-4fea-8f30-b0819cdee120.png)

**Comment by paescuj** [comment]
See https://github.com/directus/directus/discussions/10159#discussioncomment-2033583

**Reply by navanshu** [reply]
I have been through the mentioned thread, stripped prefix in nginx, the subdirectory is working, some files are loading while others are not.
Kindly check my 1st and 2nd image

**Comment by navanshu** [comment]
I solved it. It was nginx security and cache policy.
